core/src/main/java/eu/fasten/core/data/opal/MavenCoordinate.java

```diff
@@ -156,9 +156,15 @@ public String toProductUrl() {
         var repo = mavenRepos.get(0);
         return this.toURL(repo) + "/" + artifactID + "-" + versionConstraint + "." + packaging;
     }
-    
+
     @Override
     public String toString() {
-    	return getCoordinate();
+        return getCoordinate();
     }
+
+    public String toPath() {
+        return Paths.get(groupID.replace('.', '/'), artifactID, versionConstraint,
```

Comment:
This requires a unix file system. Please use the system specific separator constant.

Reply:
Stupid question, why not simply return a `Path`?

Reply:
Because, when joining paths using `Paths.get()`, `String` is needed. I've changed the method to return `Path` now.

Reply:
Addressed in [3b12da6](https://github.com/fasten-project/fasten/pull/459/commits/3b12da65de924db0fa46c81047276b27866a104a)